src/System.Runtime.Extensions/tests/System/MathF.netcoreapp1.1.cs

```diff
@@ -26,6 +26,8 @@ public static partial class MathFTests
        // use CrossPlatformMachineEpsilon * 10.
         public const float CrossPlatformMachineEpsilon = 4.76837158e-07f;
 
+        private static unsafe bool Is64Bit => sizeof(void*) == 8;
```

Comment:
The problem with `Environment.Is64BitProcess` is that it is not available prior `netstandard1.7` and since we still cross compile the test project, that won't work without having to do #ifdefs or partial classes which I think is overkill.